src/development/access-site.md

```diff
@@ -4,48 +4,124 @@ Once you have an environment running, there are many ways to access it to perfor
++-------------------------+----------------------+
+| web/sites/default/files | shared:files/files   |
+| private                 | shared:files/private |
+| tmp                     | shared:files/tmp     |
++-------------------------+----------------------+
+```
+
+The CLI also helps transferring files to and from a mount, using the `mount:upload` and `mount:download` commands. These commands use the `rsync` utility, which in turn uses SSH.
+
+For example, to download files from the 'private' mount:
+
+```
+$ platform mount:download --mount private --target ./private
+
+This will add, replace, and delete files in the local directory 'private'.
+
+Are you sure you want to continue? [Y/n]
+Downloading files from the remote mount /app/private to /Users/alice/Projects/foo/private
+  receiving file list ...   done
+
+  sent 16 bytes  received 3.73K bytes  2.50K bytes/sec
+  total size is 1.77M  speedup is 471.78
+  time: 0.91s
+The download completed successfully.
+```
+
+Uploading files to a mount is similar:
+
+```
+$ platform mount:upload --mount private --source ./private
+
+This will add, replace, and delete files in the remote mount 'private'.
+
+Are you sure you want to continue? [Y/n]
+Uploading files from /Users/alice/Projects/foo/private to the remote mount /app/private
+  building file list ...   done
+
+  sent 2.35K bytes  received 20 bytes  1.58K bytes/sec
+  total size is 1.77M  speedup is 745.09
+  time: 0.72s
+The upload completed successfully.
+```
+
+## Using SSH clients
+
+Many applications and protocols run on top of SSH, including SFTP, scp, and rsync.
+
+To obtain the SSH connection details for the environment either copy them out of the Platform.sh UI (under the "Access site" dropdown) or run:
 
 ```bash
 platform ssh --pipe
 ```
 
-That will output the connection string for SSH, including the username and host for the current project and environment.  It will look something like `<project ID>-<environment ID>--app@ssh.us.platform.sh`.  The part before the `@` is the username, the part after is the host.  Enter both of those into your SFTP client.  No password is necessary, but your SFTP client will need to have access to the SSH private key that corresponds to the public key on Platform.sh.
+That will output the connection string for SSH, including the username and host for the current project and environment.  It will look something like `<project ID>-<environment ID>--app@ssh.us.platform.sh`.  The part before the `@` is the username, the part after is the host.  Enter both of those into your SSH/SFTP client.  No password is necessary, but your client will need to have access to the SSH private key that corresponds to the public key on Platform.sh.
+
+### SFTP
+
+SFTP is another way to upload and download files to and from a remote environment. There are many SFTP clients available for every possible operating system - graphical and command-line - use whichever one works for you.
```

Comment:
"use whichever..." is its own sentence.  It should either have a period or semi-colon before it, not a dash.  Also, the dash between system and graphical should be an mdash HTML code.